Translations update from Hosted Weblate

weblate opened the pull request:
Translations update from [Hosted Weblate](https://hosted.weblate.org) for [evcc/evcc](https://hosted.weblate.org/projects/evcc/evcc/).



Current translation status:

![Weblate translation status](https://hosted.weblate.org/widget/evcc/evcc/horizontal-auto.svg)

andig commented on i18n/de.toml:
```diff
@@ -49,13 +53,13 @@ delete = "Fahrzeug löschen"
 offline = "Generisches Fahrzeug"
 online = "Fahrzeuge mit Schnittstelle"
-save = "Speichern"
+save = Speichern
```
@premultiply @naltatis warum sind einige Einträge in Anführungszeichen und viele nicht? Müssen wir da noch was am Tooling verbessern?

premultiply replied:
Das fixt mein regelmäßiger `make toml` Lauf.

andig commented on i18n/de.toml:
```diff
@@ -427,3 +431,44 @@ lineError = "In {0} wurde ein Fehler gefunden."
+current = "Aktuell"
+enabled = "Aktiviert"
+range = "Reichweite"
+soc = "SoC"
```
Ladestand?

northalpha committed:
Translated using Weblate (German)

Currently translated at 100.0% (357 of 357 strings)

Co-authored-by: northalpha <stefan+github@xn--klpping-b1a.de>
Translate-URL: https://hosted.weblate.org/projects/evcc/evcc/de/
Translation: evcc/evcc

JonasTuff committed:
Translated using Weblate (Swedish)

Currently translated at 98.3% (352 of 358 strings)

Translated using Weblate (Swedish)

Currently translated at 98.3% (351 of 357 strings)

Translated using Weblate (Swedish)

Currently translated at 96.0% (343 of 357 strings)

Co-authored-by: Jonas Gate <tidy.koala8317@brot.se>
Translate-URL: https://hosted.weblate.org/projects/evcc/evcc/sv/
Translation: evcc/evcc

RTTTC committed:
Translated using Weblate (Lithuanian)

Currently translated at 100.0% (358 of 358 strings)

Co-authored-by: RTTTC <94727758+RTTTC@users.noreply.github.com>
Translate-URL: https://hosted.weblate.org/projects/evcc/evcc/lt/
Translation: evcc/evcc

amtssp committed:
Translated using Weblate (Danish)

Currently translated at 100.0% (358 of 358 strings)

Co-authored-by: amtssp <amtssp@gmail.com>
Translate-URL: https://hosted.weblate.org/projects/evcc/evcc/da/
Translation: evcc/evcc

Commit:
Translated using Weblate (Spanish)

Currently translated at 100.0% (358 of 358 strings)

Co-authored-by: gallegonovato <fran-carro@hotmail.es>
Translate-URL: https://hosted.weblate.org/projects/evcc/evcc/es/
Translation: evcc/evcc

andig committed:
Remove guard timer

Co-authored-by: andig <cpuidle@gmx.de>

weblate committed:
Merge remote-tracking branch 'origin/master'

weblate committed:
Merge remote-tracking branch 'origin/master'

weblate committed:
Merge remote-tracking branch 'origin/master'

premultiply committed:
fix toml

premultiply committed:
Revert "Remove guard timer"

This reverts commit fa22d17acbb39f1ec11759dc44cc6dce13f64352.
